.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+    "env": {
+        "es2021": true,
+        "node": true
+    },
+    "extends": "airbnb-base",
+    "overrides": [
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "rules": {
+        "import/named": "warn",
```

Comment:
Why `warn` and not error (the default with Airbnb. I think..)

Reply:
fixed